packages/bpk-docs/src/layouts/NeoSideNavLayout/MainHeroImage.js

```diff
@@ -51,16 +51,26 @@ export default (props: Props) => {
     }
   }
 
+  const heroClassNames = getClassName(
+    'bpkdocs-main-hero-image',
+    !heroImage && 'bpkdocs-main-hero-image--collapsed',
+  );
+
+  const hamburgerMenuClassNames = getClassName(
+    'bpkdocs-main-hero-image__hamburger',
+    !heroImage && 'bpkdocs-main-hero-image__hamburger--hero-collapsed',
+  );
+
```

Comment:
I've made it so that if a hero image isn't passed in, we add new class names to change the height etc. We still want the component to render because it contains the hamburger menu and associated behaviour, which is why we just change the height.